packages/ihe-gateway/server/Channels/Default Group/Tester JSON.xml

```diff
@@ -135,33 +135,51 @@
       <transformer version="4.4.2">
         <elements/>
         <inboundDataType>JSON</inboundDataType>
-        <outboundDataType>JSON</outboundDataType>
+        <outboundDataType>XML</outboundDataType>
```

Comment:
if this was intentional, should we name the channel accordingly? ie perhaps `Tester JSON to XML` or something?

Reply:
Good catch. Likely lost this setting when pulling from server, it's supposed to be JSON throughout for this channel. Updated.